src/theme/Alert.ts

```diff
@@ -0,0 +1,85 @@
+import { AlertProps } from "@chakra-ui/react"
+
+const defaultProps = {
+  status: "info",
+  variant: "subtle",
+}
+
+const statusWarning = {
+  container: {
+    bg: "hsla(26, 96%, 65%, 10%)",
+    color: "hsla(0, 0%, 100%, 80%)",
+  },
+  icon: {
+    color: "#FB9950",
+  },
+}
+
+const statusInfo = {
+  container: {
+    color: "white",
+    bg: "#1E1E1E",
+  },
+  icon: {
+    color: "currentColor",
+  },
+}
+
+const statusSuccess = {
+  container: {
+    color: "#66FFB6",
+    bg: "#1E1E1E",
```

Comment:
Non-blocking but something to discuss in the future:

Ideally, we should put all the colors in the theme and name it. But I think this should be also specified in the designs.